libvita2d/source/vita2d_image_jpeg.c

```diff
@@ -6,23 +6,14 @@
 #include <jpeglib.h>
 #include "vita2d.h"
 
+// Max memory we want to use by jpg is 64 MB half of the VRAM, how every pixel is 4 bytes, then 64 * 1024 * 1024 / 4
+// Which is exactly the same than a picture of 4096 x 4096
+#define MAX_JPEG_MATRIX_SIZE 1662976
```

Comment:
I don't see where `1662976` comes from. `64 * 1024 * 1024 / 4` is `16777216`.

Reply:
yes sorry xD, fixing it

Reply:
Done!